Berlin/PetermannIINachtrag42.xml

```diff
@@ -48,183 +147,419 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
+                              Berlin.</q>
+                        </item>
+                        
+                        <item xml:id="e4">
+                           <locus target="#iiv"/>
+                           <desc>Note in German.</desc>
+                           <q xml:lang="de"></q>
+                        </item>
+                        
+                        <item xml:id="e5">
+                           <locus target="#1r"/>
+                           <desc>Note by a European hand.</desc>
+                        </item>
+                        
+                        <item xml:id="e6">
+                           <locus target="#1r"/>
+                           <desc>Shelfmark.</desc>
+                           <q xml:lang="de">Pet. II Nachtr. 42.</q>
+                        </item>
+                        
+                        <item xml:id="e7">
+                           <locus target="#1r"/>
+                           <desc>Stamp of the library.</desc>
+                        </item>
+                        
+                        <item xml:id="a1">
+                           <locus target="#1r"/>
+                           <desc>Note listing the main content of the manuscript.</desc>
+                           <q xml:lang="gez">ዝ፡ መጽሐፍ፡ ኢሳይያስ፡ ወኤርምያስ፡ ወባሮክ፡ ወሕዝቅኤል።</q>
+                        </item>
+                        
+                        <item xml:id="a2">
+                           <locus target="#220v"/>
+                           <desc type="Doxology">Crude note written below the end of the main text.</desc>
+                           <q xml:lang="gez">ይትባርክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ አምላከ ለኩሉ፡ መንፈስ፡ <sic>እግዚ፡</sic> ወለኵሉ፡ ዘስጋ፡</q>
+                        </item>
+                        
+                        <item xml:id="a3">
+                           <locus target="#1r"/>
+                           <desc type="OwnershipNote">Note written, probably in pencil, in the upper part of the page, partly illegible.</desc>
+                           <q xml:lang="gez">አ<gap reason="illegible" unit="chars" quantity="2"/> ፫ ለታሣስ፡ ዙ<gap reason="illegible"/> 
+                              <persName ref="PRS13996KidanaM">ኪዳነ፡ ማርያም፡</persName></q>
+                        </item>
+                        
+                        <item xml:id="a4">
+                           <desc type="Correction">Erasures and corrections by a more recent hand in a few places.</desc>
+                           <q xml:lang="gez"/>
+                        </item>
+                        
+                        <item xml:id="a5">
+                           <desc type="Comment">A more recent hand has noted the European chapter numbers 
+                              (<foreign xml:lang="gez">ምዕራፍ፡</foreign>) in the margins, denotating their place in the text with
+                              a cross and a mark within the text.</desc>
```

Comment:
I was curious about what the "mark" that is not a cross was… and I looked quickly at the first 30 pages and saw crosses only in a few cases – most often, the additional chapter markings seem to coincide with a punctuation mark or rubricated element etc., which in itself seems to have been enough to mark where the chapter started. To cut it short: maybe one could add a "sometimes" before "detonating…". And are there really crosses PLUS additional other marks?

Reply:
I meant cases such as this (f. 110v):
![image](https://github.com/BetaMasaheft/Manuscripts/assets/19650382/b345d7b7-6f5b-4973-b8c2-0147f7c6862b)
I changed the formulation to be more general.